bower.json

```diff
@@ -35,9 +35,9 @@
   ],
   "dependencies": {
     "firebase": "2.1.x",
-    "ember": "1.9.x",
     "ember-data": "1.0.0-beta.12"
   },
   "devDependencies": {
+    "ember": "1.9.x"
```

Comment:
Is including `ember` as a dev dependency even required? If `ember-data` will include a copy of `ember` itself, why do we even need to list it at all? It just seems weird to me that `ember` would be a dev dependency to EmberFire.